js/commerce_bluesnap.form.js

```diff
@@ -0,0 +1,252 @@
+/**
+ * @file
+ * Javascript to submit card details to BlueSnap in a PCI-compliant way.
+ */
+
+(function ($, Drupal, drupalSettings) {
+
+  'use strict';
+
+  /**
+   * Attaches the commerceBluesnapForm behavior.
+   *
+   * @type {Drupal~behavior}
+   *
+   * @prop {Drupal~behaviorAttach} attach
+   *   Attaches the commerceBluesnapForm behavior.
+   * @prop {Drupal~behaviorDetach} detach
+   *   Detaches the commerceBluesnapForm behavior.
+   *
+   * @see Drupal.commerceBluesnap
```

Comment:
**CHANGE REQUEST**

So this annotation does not have any meaning because `Drupal.commerceBluesnap` does not currently exist. `@see Drupal.commerceStripe` is set on that module because they store additional settings e.g. it most likely refers to `drupalSettings.commerceStripe.publishable_key`.

I see you use `drupalSettings.bluesnap_token`, it might be good to change that to `drupalSettings.commerceBluesnap.token` and keep the annotation here. It's always good to namespace variables per module. That should be enough for now, but we'll need to keep in mind that we will have more gateways that may have their own settings. The user can switch between payment methods and that may attach more variables on the same global JS object. The point is that eventually we may need `drupalSettings.commerceBluesnap.hostedPaymentFields.token` and `drupalSettings.commerceBluesnap.embeddedCheckout.token` without conflicts.

We can keep things to `drupalSettings.commerceBluesnap.token` and keep this in mind when we develop the remaining payment methods.